auth-web/src/views/auth/AcceptInviteLandingView.vue

```diff
@@ -44,8 +55,10 @@ export default class AcceptInviteLandingView extends Vue {
   @Prop() token: string
 
   private otherError: boolean = false
+  private isCreateAccount: boolean = false
 
   private mounted () {
+    this.isCreateAccount = (this.$route?.name === 'createuserprofile')
```

Comment:
Add a constant again for this createuserprofile ;I think we have a pages constant